<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF目录生成器</string>
  </property>
  <widget class="PushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>导入PDF文件</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="PushButton" name="pushButton_2">
        <property name="text">
         <string>提取图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButton" name="pushButton_3">
        <property name="text">
         <string>OCR识别</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="PushButton" name="pushButton_4">
        <property name="text">
         <string>书签写入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButton" name="pushButton_5">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="PushButton" name="pushButton_6">
        <property name="text">
         <string>一键完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButton" name="pushButton_7">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>导出路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="LineEdit" name="lineEdit_4">
      <property name="text">
       <string>./result/out/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PushButton" name="pushButton_8">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>目录起始页：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="LineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>目录结束页：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="LineEdit" name="lineEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>正文开始页：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="LineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="RadioButton" name="radioButton">
      <property name="text">
       <string>单栏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="RadioButton" name="radioButton_2">
      <property name="text">
       <string>双栏</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>445</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>目录格式</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>831</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>PDF路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="LineEdit" name="lineEdit_5"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
